device-types/FS/S5860-20SQ.yaml

```diff
@@ -0,0 +1,73 @@
+---
+manufacturer: FS
+model: S5860-20SQ
+slug: s5860-20sq
+part_number: s5860-20sq
+u_height: 1
+is_full_depth: true
+airflow: front-to-rear
+console-ports:
+  - name: Management
+    type: rj-45
+power-ports:
+  - name: Slot 1
+    type: iec-60320-c14
+    maximum_draw: 85
+    allocated_draw: 85
+  - name: Slot 2
+    type: iec-60320-c14
+    maximum_draw: 85
+    allocated_draw: 85
+interfaces:
+  - name: Te1/1
```

Comment:
I do not work with these devices so I am just confirming, in the configuration are these referenced as `Te1/1` or do they have a full name such as in Cisco IOS devices `GigabitEthernet1/0/1`

Reply:
If they are actually `Te1/1` feel free let me know and resolve this

Reply:
My bad, just checked on an unit and in fact they are _fullinterfacename_ _unit/slot/port_

Reply:
Fixed on #57fe0051cb2bad4312d590ce8a4d569e7e321d19